docker/Dockerfile.ci-mirror

```
@@ -0,0 +1,86 @@
+        ca-certificates curl git unzip xz-utils gnupg \
+        libgtk-3-0t64 libgbm1 libxt6 libdbus-glib-1-2 \
+        libasound2t64 libnss3 libxss1 libxrandr2 \
+        libpangocairo-1.0-0 libatk1.0-0t64 libatk-bridge2.0-0t64 \
+        libcups2t64 libxcomposite1 libxdamage1 libxkbcommon0 \
+        fonts-liberation libdrm2 libxshmfence1 \
+        # gh CLI for Camoufox fallback download path
+        wget \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install GitHub CLI (gh) — Camoufox install-action's fallback uses it.
+RUN curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)